plugins/destination/duckdb/client/client_test.go

```diff
@@ -40,6 +49,9 @@ func TestPlugin(t *testing.T) {
 			SkipIntervals: true,
 			// not supported in duckDB for now
 			SkipLargeTypes: true,
+			// not supported in Appender:
+			SkipMaps:    true,
+			SkipStructs: true,
```

Comment:
I wonder if we can remove this & use string/JSON?

Reply:
Struct as JSON done in https://github.com/cloudquery/cloudquery/pull/16668/commits/ecaf3a261cd96eaf8cf0e62cc7b80003674c5553 will now work on maps

Reply:
Maps as JSON done in https://github.com/cloudquery/cloudquery/pull/16668/commits/6eaf9da7f2ad42923ed578d05531230276168661 PTAL